docs/release.rst

```diff
@@ -314,6 +314,12 @@ Then rebuild everything that is templatized:
 
 This should set up infrastructure and some other parts like makefile variables and pykickstart version used.
 
+Lastly it is necessary to set up updated l10n commit hash - check the commit hash of the ``anaconda-l10n`` repo,
+the one where the new f<version> folder has been added and put the hash to the ``GIT_L10N_SHA`` variable in the
+``po/l10n-config.mk`` file.
+
+This is necessary for the Web UI related translation pinning to work & l10n branching checks to pass.
+
```

Comment:
This is not necessary to be done manually. Only case is if you doing release before the `anaconda-l10n` automation is started. At least I would expect that.

This is definitely not something we want to do manually - copying hashes.

Reply:
If you keep the hash the same, it will fail the branching check, because the directory is literally missing. Also all subsequent rpm builds etc.... you get the idea.